src/client/VZSidebar/Search.tsx

```diff
@@ -159,48 +305,68 @@ export const Search = () => {
                         match.index + pattern.length,
                       );
 
+                      const identifier = file.name + "-" + match.line;
```

Comment:
Ideally we shouldn't need this if we are using `useRef` to get at the DOM elements.